docs/backend-requests/making/jwt-templates.mdx

```diff
@@ -57,15 +57,27 @@ A token generated using the template above, would look roughly like this:
 
 In the previous example, "automatically-included claims" was briefly mentioned. Besides the claims specified by the template, there are certain claims that are automatically included in all tokens and cannot be overridden by templates. Clerk calls these "default claims" and they are the following:
 
-- `sub`: The user's unique ID (e.g. `user_abc1234def57`)
-- `iat`: The time at which the token was issued, as a Unix timestamp (e.g. `1516239022`)
-- `iss`: The Frontend API URL of your instance (e.g. `https://clerk.your-site.com`)
-- `jti`: A random alphanumeric string, used as a unique identifier for the token
-- `exp`: The time after which the token will expire, as a Unix timestamp. Determined using the **Token lifetime** template setting.
-- `nbf`: The time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** template setting.
-- `azp`: The origin of the request. If it's not present, this claim is omitted.
+* `azp`: authorized party - the `Origin` header that was included in the original Frontend API request made from the user. Most commonly, it will be the URL of the application (e.g. `https://example.com`).
+* `exp`: expiration time - the time after which the token will expire, as a Unix timestamp. Determined using the **Token lifetime** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+* `iat`: issued at - the time at which the token was issued, as a Unix timestamp (e.g. `1516239022`).
+* `iss`: issuer - the Frontend API URL of your instance (e.g. `https://clerk.your-site.com`).
+* `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
+* `sid`: session ID - the ID of the current session.
+* `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
+* `orgs`: organizations - the organizations that the user belongs to.
+* `org_id`: organization ID - the ID of the organization that the user belongs to.
+* `org_slug`: organization slug - the slug of the organization that the user belongs to.
+* `org_role`: organization role - the role of the user in the organization.
```

Comment:
While `org_x` exists in session JWT tokens, they don't in JWT Templates. Also, `orgs` is deprecated, so this won't be returned for new applications in either.